GOVERNANCE.md

```diff
@@ -0,0 +1,44 @@
+NOTE: The following is a snapshot of a perpetually changing state of affairs in
+the WGPU community. It is not a binding contract between users and
+decision-makers of the WGPU project.
+
+---
+
+The **WGPU project** is an ecosystem that _enables application authors to write
+portable and performant graphics programs_. It was originally conceived to
+provide an implementation of WebGPU for Firefox as the standard evolved and
+settled into something that could be shipped on all web browsers. WGPU has also
+enjoyed much contribution and use from other projects that requires graphics
+programming. We expect that these sorts of users will continue for the lifetime
+of project, and we embrace these contributors' needs and effort as the lifeblood
+of WGPU.
+
+## Mission
+
+The WGPU community seeks to realize the following directives through the
+project: it…
+
+1. …provides an implementation of the WebGPU API that is…
+    1. …correct and fully conformant.
+    1. …portable across all major platforms.
+    1. …performant enough to enable demanding applications.
+1. …serves as a platform of experimentation for:
+    1. …WebGPU standards development.
+    1. …native application authors.
```

Comment:
Very happy with the first point, but I feel a few important aspects of wgpu are not quite captured yet here:
* seamless use as a graphics backend in applications targeting **both native & web**
* provide a WebGPU _style_ graphics API abstraction to native _Rust_ applications 

currently 'native' is only under the "platform of experimentation" which sounds a bit like anything that isn't part of a WebGPU implementation isn't fully supported.
I'm still pondering a bit how much our many many native extensions are directly part of the mission: they're clearly in part (!) experimentation but realistically this is far outside of what can be expected to be part of WebGPU any time soon. But these are still important pieces in providing a flexible WebGPU-like api to native Rust users, so I believe this deserves a place here.
My understanding of wgpu's mission is first WebGPU impl (as already well described above), then native graphics abstraction library with seamless web support, then platform for experimentation as described. All of these are obviously interconnected (well otherwise we wouldn't put it into a single family of libraries), but they do occasionally pull into different directions and it's worth pointing them and the implicit hierarchy out

Reply:
I've attempted a revision of this text along the lines of what it seems you want at cd8cf89d5, but I'm not confident I've captured things to your satisfaction. Whaddaya think?

Reply:
nice, has all the points baked in, I like it. 🚢